distribution/lib/Standard/Table/0.0.0-dev/src/Data_Formatter.enso

```diff
@@ -13,7 +13,7 @@ import project.Value_Type.Auto
 import project.Value_Type.Bits
 import project.Value_Type.Value_Type
 
-polyglot java import java.lang.Exception as Java_Exception
+polyglot java import java.lang.Exception as JException
```

Comment:
Just trying to understand: what was wrong with `Java_Exception` name?

Reply:
Makes it easier to catalog all the catches in one regex.

Reply:
Can't we have an analogous regex with `Java_Exception`?

`Panic.catch (Any|Java_Exception)`

Reply:
yes just easier to have one name across the codebase - no actual big reason.